lib/rubocop/version.rb

```diff
@@ -63,9 +63,21 @@ def self.parser_version
     end
 
     # @api private
+    # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity
     def self.extension_versions(env)
+      plugins = config_for_pwd(env).loaded_plugins
+      plugin_versions = plugins.filter_map do |plugin|
+        next if Plugin::BUILTIN_INTERNAL_PLUGINS.key?(plugin.about.name)
```

Comment:
Perhaps we could extract a `plugin.internal?` method?

Reply:
Unfortunately, the `LintRoller::Plugin` class does not have such a method, so it is not possible to achieve this with the current basic functionality.  
https://github.com/standardrb/lint_roller/blob/v1.1.0/lib/lint_roller/plugin.rb

Although it could be done by implementing some form of extension, including a custom one, it does not seem necessary to go that far at this stage just for this logic.